extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessorUtil.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.kubernetes.deployment;
+
+import static io.quarkus.kubernetes.deployment.Constants.GITOPS_DIRECTORY;
+import static io.quarkus.kubernetes.deployment.Constants.GITOPS_FRIENDLY;
+
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+public class KubernetesProcessorUtil {
+
+    public static boolean makeGitOpsFriendly() {
+        Config config = ConfigProvider.getConfig();
+        String configValue = config.getOptionalValue(GITOPS_FRIENDLY, String.class)
+                .orElse("true")
```

Comment:
If you define a default value of `true` you don't need an optional

Reply:
Where would be the proper place to define this default value?